database/etcd.go

```diff
@@ -0,0 +1,80 @@
+package database
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/gobackup/gobackup/helper"
+	"github.com/gobackup/gobackup/logger"
+)
+
+// etcd database
+//
+// ref:
+// https://etcd.io/docs/v3.4/dev-guide/interacting_v3/
+//
+// # keys
+//
+//   - type: etcd
+//   - endpoints: [localhost]
+//   - port: 2379
+//   - user:
+//   - password:
+//   - cacert:
+//   - cert:
+//   - key:
+//   - insecure-skip-tls-verify: false
+//   - args:
+type Etcd struct {
+	Base
+	endpoints     []string
+	args          string
+	_dumpFilePath string
```

Comment:
Em...

Sorry, again, why do you removed the `username` and `password`.

You can follow this config.

- endpoints
- username
- password
- skip_verify 

https://gobackup.github.io/configuration/databases/influxdb

Reply:
Or is the `user` and `password` is also optional in ectd?

Reply:
You can choose whether to enable authentication in etcd. By default, it's disabled, and anyone can access the data. If you don't enable authentication, username and password become irrelevant.

Reply:
Obey, I got it.